adafruit_minimqtt/matcher.py

```diff
@@ -27,22 +34,22 @@ class Node:
 
         __slots__ = "children", "content"
 
-        def __init__(self):
-            self.children = {}
+        def __init__(self) -> None:
+            self.children: Dict[str, MQTTMatcher.Node] = {}
             self.content = None
 
-    def __init__(self):
+    def __init__(self) -> None:
         self._root = self.Node()
 
-    def __setitem__(self, key, value):
+    def __setitem__(self, key: str, value: Callable[..., Any]) -> None:
```

Comment:
Since it's `Callable[..., Any]`, it can be shortened to just `Callable`.

Reply:
`mypy` does not like bare Callable:
```
Missing type parameters for generic type "Callable"
```
Also, I was thinking that with the future type hint additions to the main module the types of the arguments can be back filled.

Reply:
On second thought, let's leave that unannotated for now so it doesn't look mistakenly annotated.  it may even be a good idea to leave a commented to-do so that when annotations are finished this should be revisited.

Reply:
okay, Callable removed. Will add a note to the issue tracking the type hints.